xarray/core/variable.py

```diff
@@ -1091,6 +1092,10 @@ def chunk(self, chunks={}, name=None, lock=False):
 
             data = da.from_array(data, chunks, name=name, lock=lock, **kwargs)
 
+        # rechunking erases encoding
+        if self._encoding and "chunks" in self._encoding:
+            del self._encoding["chunks"]
```

Comment:
we should mention this in the docstring for `DataArray.chunk` `Dataset.chunk` and `Variable.chunk`